ontology.ttl

```diff
@@ -1,469 +1,1370 @@
-
-:PublishedStatus a rdfs:Class ;
-  rdfs:subClassOf :Status ;
-  rdfs:comment "The status of a published Publication" .
+nva: a owl:Ontology ;
+  dcterms:title "The NVA Core ontology" ;
+  dcterms:description "The terminology definitions used in the APIs used in NVA" ;
+  dcterms:creator "Unit" ;
+  dcterms:publisher "Unit" ;
+  dcterms:license <https://creativecommons.org/publicdomain/zero/1.0> .
+
+nva:Resource a rdfs:Class ;
+  rdfs:isDefinedBy nva: ;
+  rdfs:label "Resource" ;
+  rdfs:comment "A thing in NVA" .
+
+nva:Publication a rdfs:Class ;
+  owl:sameAs nva:Resource .
+
+nva:Agent a rdfs:Class ;
+  rdfs:isDefinedBy nva: ;
+  rdfs:label "Agent" ;
+  rdfs:comment "A person or a group of people" .
+
+nva:Organization a rdfs:Class ;
+  rdfs:isDefinedBy nva: ;
+  rdfs:label "Organization" ;
+  rdfs:subClassOf nva:Agent ;
+  rdfs:comment "A group of people forming an administratively coherent group" .
+
+# Unlikely represented in our model directly, but I think it should be
+
+nva:Person a rdfs:Class ;
+  rdfs:isDefinedBy nva: ;
+  rdfs:label "Person" ;
+  rdfs:subClassOf nva:Agent ;
+  rdfs:comment "An individual human being" .
+
+nva:Time a rdfs:Class ;
+  rdfs:isDefinedBy nva: ;
+  rdfs:label "Time" ;
+  rdfs:comment "A instant or a period defined in relation to a common temporal scheme" .
+
+nva:Event a rdfs:Class ;
+  rdfs:isDefinedBy nva: ;
+  rdfs:label "Event" ;
+  rdfs:comment "A thing that has or will occur" .
+
+nva:VirtualEvent a rdfs:Class ;
+  rdfs:isDefinedBy nva: ;
+  rdfs:subClassOf nva:Event ;
+  rdfs:label "VirtualEvent" ;
+  rdfs:comment "A thing that has or will occur in a virtual forum" .
+
+nva:Meeting a rdfs:Class ;
+  rdfs:subClassOf nva:Event ;
+  rdfs:isDefinedBy nva: ;
+  rdfs:label "Meeting" ;
+  rdfs:comment "A gathering of people designed to discuss a common topic or set of topics" .
+
+nva:Place a rdfs:Class ;
+  rdfs:isDefinedBy nva: ;
+  rdfs:label "Place" ;
+  rdfs:comment "A geographically located thing" .
+
+nva:DoiRequest a rdfs:Class ;
+  rdfs:isDefinedBy nva: ;
+  rdfs:label "DoiRequest" ;
+  rdfs:comment "A request event for a DOI" .
+
+nva:EntityDescription a rdfs:Class ;
+  rdfs:isDefinedBy nva: ;
+  rdfs:label "EntityDescription" ;
+  rdfs:comment "A collection of metadata associated with a Resource" .
+
+nva:PublicationDate a rdfs:Class ;
+  rdfs:isDefinedBy nva: ;
+  rdfs:label "PublicationDate" ;
+  rdfs:comment "A temporal instant" .
+
+nva:Contributor a rdfs:Class ; # Should be renamed to "Contribution"
+  rdfs:isDefinedBy nva: ;
+  rdfs:label "Contributor" ;
+  rdfs:comment "A participation event in the creation of a resource" .
+
+# The following three classes are problematic since they are subclassed in from an unnecessary class
+# Person should be added and Identity, Personal, Organizational should be removed
+
+nva:Identity a rdfs:Class ;
+  rdfs:isDefinedBy nva: ;
+  rdfs:label "Identity" ;
+  rdfs:comment "A person or organization that has responsibility for a contribution event" .
```

Comment:
I am not sure about the clarity of the meaning. I understand that this is the entity that makes the actual contribution. Who is responsible for it, is a bit unclear. If, for instance, I am a Ph.D. creating an article, am I the sole responsible for its content?